src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -309,20 +309,29 @@ public void InitializeService(ServiceHost serviceHost, SqlToolsContext context)
             await Task.FromResult(true);
         }
 
-        private static async Task HandleSignatureHelpRequest(
+        private static async Task HandleDocumentHighlightRequest(
             TextDocumentPosition textDocumentPosition,
-            RequestContext<SignatureHelp> requestContext)
+            RequestContext<DocumentHighlight[]> requestContext)
         {
             await Task.FromResult(true);
         }
+#endif
 
-        private static async Task HandleDocumentHighlightRequest(
+        private static async Task HandleSignatureHelpRequest(
             TextDocumentPosition textDocumentPosition,
-            RequestContext<DocumentHighlight[]> requestContext)
+            RequestContext<SignatureHelp> requestContext)
         {
-            await Task.FromResult(true);
+            var scriptFile = WorkspaceService<SqlToolsSettings>.Instance.Workspace.GetFile(
```

Comment:
Best practice is not to use var in source if the right-hand side isn't clearly typed. Here, I'm wondering is this a File, FileInfo, or some different type of object. Could you use full names for this and the "var help"?